examples/elasticache/v1beta1/replicationgroup-engineVersion.yaml

```
@@ -0,0 +1,19 @@
+apiVersion: elasticache.aws.upbound.io/v1beta1
+kind: ReplicationGroup
+metadata:
+  annotations:
+    meta.upbound.io/example-id: elasticache/v1beta1/replicationgroup
+    uptest.upbound.io/timeout: "3600"
+  labels:
+    testing.upbound.io/example-name: example
+  name: example
```

It would be better if we randomize the `metadata.name` fields.

```suggestion
  name: example-${Rand.RFC1123Subdomain}
```

---

Good idea! 👍